docs/Learning-Environment-Examples.md

```diff
@@ -82,16 +82,16 @@ you would like to contribute environments, please see our
 
 ![GridWorld](images/gridworld.png)
```

Comment:
Possible to link to this environment in the goal signal docs and the Changelog? Just in case a user wants an example of how to use these features